pkg/vulnsrc/k8svulndb/testdata/happy-fixed/vuln-list-k8s/upstream/CVE-2023-2727.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "CVE-2023-2727",
+  "modified": "2023-06-13T14:42:06Z",
+  "published": "2023-06-13T14:42:06Z",
+  "summary": "Bypassing policies imposed by the ImagePolicyWebhook and bypassing mountable secrets policy imposed by the ServiceAccount admission plugin",
+  "details": "Users may be able to launch containers using images that are restricted by ImagePolicyWebhook when using ephemeral containers. Kubernetes clusters are only affected if the ImagePolicyWebhook admission plugin is used together with ephemeral containers.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS:3.1",
```

Comment:
Here we are using `CVSS_V3` to decode severity - https://github.com/aquasecurity/trivy-db/blob/0155e8b2d9e3077da1e72db5e800c7138ae8926a/pkg/vulnsrc/osv/osv.go#L318

If i understand correctly - this is a typo (https://github.com/aquasecurity/vuln-list-k8s/pull/92/files#diff-bfaecf1529a98db9d6d59c17ddff6f603038d988190ae591c51fd58a494ace17R15). Can you change this to see severity in tests?